admin-dev/themes/default/template/controllers/attribute_generator/content.tpl

```diff
@@ -135,7 +135,7 @@
 						</tr>
 					</tbody>
 				</table>
-				<div class="alert alert-info">{l s='Please click on "Generate these combinations".'}</div>
+				<div class="alert alert-info">{l s='Please click on "%generate_label%".' sprintf=['%generate_label%' => {l s='Generate these combinations'}]}</div>
```

Comment:
No Domain ?

Reply:
No this one is in `messages` and never had a domain defined It could be nice to add one but the thing the whole file has no domain so it would grow the PR scope quite a lot